### PR TITLE
Integration test k3d runtimes

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: runtime-k3d/test/app/requirements-links.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+git+https://github.com/eclipse-velocitas/vehicle-app-python-sdk.git@v0.9.0
```
Comment (MP91) [inline runtime-k3d/test/app/requirements-links.txt]:
```suggestion
git+https://github.com/eclipse-velocitas/vehicle-app-python-sdk.git@v0.9.1
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -118,3 +118,49 @@ jobs:
...
+
+      - name: Set up Homebrew
+        id: set-up-homebrew
+        uses: Homebrew/actions/setup-homebrew@master
+
+      - name: Install required packages
+        run: |
+          sudo curl -L https://github.com/eclipse-velocitas/cli/releases/latest/download/velocitas-linux-x64 -o /usr/bin/velocitas
+          sudo chmod +x /usr/bin/velocitas
+          brew install k3d dapr/tap/dapr-cli
```
Comment (MP91) [inline .github/workflows/ci.yml]: Please pin the dapr version to the one we use in the template

Comment (MP91) [reply .github/workflows/ci.yml]: It is also fine to go back to the binaries if needed, since it is also way faster

**Review by MP91 — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -118,3 +118,49 @@ jobs:
...
+          cd runtime-k3d/test
+          pip install -r requirements.txt
+          pytest -s ./integration/integration_test.py
+
+      - name: Upload log files as artifacts
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          path: |
+            ./runtime-k3d/test/logs/runtime-k3d
```
Comment (MP91) [inline .github/workflows/ci.yml]: Please add a proper name for the artifact

**Review by MP91 — APPROVED** [review]
LGTM

**Review by dennismeister93 — APPROVED** [review]
LGTM! :)

**Pull request merged**